doc/source/docsportal_sec.rst.inc

```diff
@@ -0,0 +1,199 @@
+    directly to the web or behind additional reverse proxy implementing 
+    required security limitations.
+
+* Storage
+
+  * OpenStack Swift storage. Practically this can be any other object storage
+    which allows web access.
+
+  * Zuul CI/CD writes approved content into the storage
+    destination.
+
+  * Web Server fetches the content.
+
+  * The content in the Storage for the Documentation Portal is by
+    definition public content (not protected by any additional ACLs).
+
+  * If the content is not designed to be publicly available content in the
+    Storage must be protected by ACLs. This in turn will require enabling web
+    server to access this content (i.e. swift-proxy in the case of using
+    OpenStack Swift).
+
+* :ref:`zuul`
+
+  * Zuul installation manages git projects and implements
+    configured CI rules in order to ensure that only checked and
+    approved content will be merged. Default configuration
+    forbids anybody (except of Zuul Administrators) to bypass
+    required checks and merge content manually
+  * Once all the prerequisites are fulfilled Zuul merges Pull
+    Request, builds documentation and pushes it to storage with
+    dedicated credentials.
+  * Only git projects explicitly included in the Zuul tenant are
+    being respected. Registered git projects with disabled
+    branch protection rules are ignored.
+
+* `GitHub <https://github.com>`_
+
+  * An external git hosting provider.
+  * Projects in the GitHub organization are managed by `dedicate process
+    <https://docs.otc-service.com/system-config/gitcontrol.html>`_
+
+Interface Description
+---------------------
+
+The only public facing interface is the regular Web using HTTPS (automatic
+forwarding from HTTP).
+
+Tenant Security
+---------------
+
+Documentation Portal is not providing any tenant support. All documents that are hosted
+on the Help Center are placed in a dedicated storage (as public content). User
+management is also not supported.
+
+O&M Access Control
+------------------
+
+Only users enabled in the :git_file:`inventory/base/group_vars/all.yaml` are
+able to login to the underlaying infrastructure. Direct access to the hosts is
+only possible through the :ref:`Bridge` host which serves as a bastion host.
+
+Logging and Monitoring
+----------------------
+
+Every component of the HelpCenter produces own logs.
+
+* haproxy log (VM service logs)
+* nginx log (VM or Kubernetes POD log)
+* Swift proxy and storage service logs
+* Zuul logs
+
+  * public job logs (test build log file)
+  * executor log
+  * scheduler log
+
+Patch Management
+----------------
+
+The service consists of OpenSource elements only. Whenever new release of any
+software element (haproxy, nginx, zuul) is identified a Pull Request to this
+repository need to be created to update the software. Pathing of the
+underlaying VM (haproxy) is executed as a regular job applying all the existing
+OS updates.
+
+Hardening
```

Comment:
Maybe some reference to Deutsche Telekom Hardening policies?

Reply:
Added